applications/GeoMechanicsApplication/custom_elements/U_Pw_updated_lagrangian_element.cpp

```diff
@@ -168,7 +98,11 @@ void UPwUpdatedLagrangianElement<TDim, TNumNodes>::CalculateOnIntegrationPoints(
     }
 }
 
-//----------------------------------------------------------------------------------------
+template <unsigned int TDim, unsigned int TNumNodes>
```

Comment:
Not sure if I understand why this function that returns an empty vector must be created, at the place where its used, we then again test for empty.

Reply:
The problem is that we want to re-use the calculate all of the base class, but use different permeability factors than for the base class (namely no update factors, since that's in the geometrically non-linear behavior). Therefore we need a virtual function that either returns the values, or empty. An alternative would be to return an array of 1.0 (but then we get a redundant multiplication).